Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -718,20 +735,31 @@ public struct ApolloCodegenConfiguration: Codable, Equatable {
     /// cases on the generated Swift enums.
     /// Defaultss to ``ApolloCodegenConfiguration/CaseConversionStrategy/camelCase``
     public let enumCases: CaseConversionStrategy
+    
+    /// Determines how the names of fields in the GraphQL schema will be converted into
+    /// properties in the generated Swift code.
+    /// Defaultss to ``ApolloCodegenConfiguration/CaseConversionStrategy/camelCase``
+    public let fieldCasing: CaseConversionStrategy
 
     /// Default property values
     public struct Default {
       public static let enumCases: CaseConversionStrategy = .camelCase
```

Comment:
I don't think it's right to have an enum case called `default` if it's not actually the default value for all the configurations.

Along with the fact that the `default` behavior is different for `enumCases` and `fieldCasing`, it really feels like a bad design choice to me. Which is leading me back to thinking we should bite the bullet and just go with two separate enums.